Census/Migrations/20240408185717_InitialCreate.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace LantanaGroup.Link.Census.Migrations
+{
+    /// <inheritdoc />
+    public partial class InitialCreate : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.CreateTable(
+                name: "CensusConfig",
+                columns: table => new
+                {
+                    Id = table.Column<string>(type: "nvarchar(450)", nullable: false),
```

Comment:
One thing I did different in Tenant was to overwrite the BaseEntity from Shared and replace String with Guid so that column gets generated with type uniqueIdentifier versus varchar like:
CREATE TABLE [Facilities] (
     [Id] uniqueidentifier NOT NULL,